tests/test_browser.py

```diff
@@ -2107,6 +2107,16 @@ def test_sdl2_image_jpeg(self):
     ]).communicate()
     self.run_browser('page.html', '', '/report_result?600')
 
+  def test_sdl2_image_formats(self):
+    shutil.copyfile(path_from_root('tests', 'screenshot.png'), os.path.join(self.get_dir(), 'screenshot.not'))
+    open(os.path.join(self.get_dir(), 'sdl2_image_formats.c'), 'w').write(self.with_report_result(open(path_from_root('tests', 'sdl2_image.c')).read()))
```

Comment:
instead of `with_report_result` etc., can just use `btest` which is a lot simpler.